cluster-autoscaler/cloudprovider/packet/packet_cloud_provider.go

```diff
@@ -128,51 +136,51 @@ func (pcp *packetCloudProvider) NodeGroupForNode(node *apiv1.Node) (cloudprovide
 }
 
 // HasInstance returns whether a given node has a corresponding instance in this cloud provider
-func (pcp *packetCloudProvider) HasInstance(node *apiv1.Node) (bool, error) {
+func (pcp *equinixMetalCloudProvider) HasInstance(node *apiv1.Node) (bool, error) {
```

Comment:
I'm curious what we are missing out on by not implemented these methods where it seems we could easily offer a meaningful lookup:
- HasInstance
- GetAvailableMachineTypes

Also for these methods that seem more important than lookups:
- Refresh
- Cleanup

Reply:
(and any others that are "NotImplemented" / "not implemented"